solr/core/src/test/org/apache/solr/core/TestMergePolicyConfig.java

```diff
@@ -88,7 +88,7 @@ public void testLegacyMergePolicyConfig() throws Exception {
     TieredMergePolicy tieredMP = assertAndCast(TieredMergePolicy.class, iwc.getMergePolicy());
 
     assertEquals(10, tieredMP.getMaxMergeAtOnce());
-    assertEquals(10.0D, tieredMP.getSegmentsPerTier(), 0.0D);
+    assertEquals(8.0D, tieredMP.getSegmentsPerTier(), 0.0D);
```

Comment:
https://github.com/apache/lucene/pull/14823 - wondering if this is worth mentioning in the upgrade notes so that anyone wishing to retain the existing value can go and configure it.

Reply:
https://github.com/apache/solr/pull/3659/commits/112844905d32d8c87412c67f6b700ef6c2e4f1b3 for that.